libraries/config/include/config/hardfork.hpp

```diff
@@ -26,6 +26,8 @@ void dec_json(const Json::Value& json, MagnoliaHardfork& obj);
 struct HardforksConfig {
   // disable it by default (set to max uint64)
   uint64_t fix_redelegate_block_num = -1;
+  // disable it by default (set to max uint64)
+  uint64_t phalaenopsis_hf_block_num = -1;
```

Comment:
I think it should be enabled by default `= 0`

Reply:
This change will affect only some default networks that won't have this value in predefined genesis. So maybe we can discuss this on Monday meet?

Reply:
yeah we can, but there is no reason for any network to have this off... Most of the hardforks should be enabled by default